examples/src/main/scala/za/co/absa/spline/issue/GitHub738Job.scala

```diff
@@ -33,8 +33,8 @@ object GitHub738Job extends SparkApp("GitHub spline-738") with Logging {
   val primarySourcePaths = primarySourceKeys.map(key => createMockCSV(s"Primary-$key"))
```

Comment:
It would be nice to have a short comment on this job, it's good to have a ticket number, but still...

the comment could describe what is the purpose of this job and link to the ticket, since it's not in agent github, so it may be otherwise confusing.

Reply:
It's in Spline's repo, the old one - AbsaOSS/spline#738

Reply:
done. please check